map_type: costmap
track_unknown_space: true
#unknown_threshold: 9
transform_tolerance: 0.3
obstacle_range: 2.5
raytrace_range: 3.0
#robot_radius: 0.2
footprint: [[0.29, 0.25], [0.29, -0.25], [-0.29, -0.25], [-0.29, 0.25]]
#footprint: [[0.30, 0.30], [0.30, -0.30], [-0.30, -0.30], [-0.30, 0.30]]
inflation_radius: 0.60
                       
observation_sources: base_scan
base_scan: {sensor_frame: base_laser, data_type: LaserScan, topic: /scan, expected_update_rate: 0.2, obstacle_range: 2.0, raytrace_range: 2.5,
  observation_persistence: 0.0, marking: true, clearing: true, max_obstacle_height: 0.5, min_obstacle_height: 0.0}

NavfnROS:
  allow_unknown: true

  
  

